README.md

```diff
@@ -1,5 +1,5 @@
 [one-app-bundler]: https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-bundler
-[React]: http://reactjs.org/
+[react]: http://reactjs.org/
```

Comment:
```suggestion
[React]: http://reactjs.org/
```

Reply:
Good catch, line 13 in the `README` needs this capitalized to link:
```
server that serves up a single page app built using [React] components and
```

Reply:
Resolved in fd28082